### DISCUSSION TITLE
Is there any option to build custom watermark for the document?

### DISCUSSION THREAD

**Discussion by jothibasu-smartaudit** [dolanmiu/docx#1480] [Q&A]
![Screenshot 2022-04-14 at 10 43 20 AM](https://user-images.githubusercontent.com/55376342/163318414-e06b46ee-6a83-4d8b-8e70-87918b9ca429.png)

**Comment by MasterBran** [comment]
I found a solution in #338 , but not work, so did you find any solution?

**Reply by jothibasu-smartaudit** [reply]
Tried the same solution initially.
But it doesn't work as expected, since it's just the tweak.

So, not using the watermark option now.
Looking for something else. Will let you know once I found the right one.

**Reply by BaoHP** [reply]
Do you found any way for this watermark ?

**Reply by jothibasu-smartaudit** [reply]
I've stopped working on this
